stytch/consumer/api/sessions.py

```diff
@@ -258,7 +259,7 @@ def authenticate_jwt(
         session_jwt: str,
```

Comment:
I can't comment any higher, but `# ADDIMPORT: from stytch.consumer.models.sessions import AuthenticateJWTLocalResponse` or it'll get removed the next time codegen runs